### DISCUSSION TITLE
柱状图能不能根据类型设置不同柱子宽度？

### DISCUSSION THREAD

**Discussion by SplitterChan** [antvis/G2Plot#3562] [Q&A]

**Comment by hustcc** [comment]
有更详细的描述吗？

**Reply by wangyue-gagua** [reply]
G2中可以通过chart.interval().size()来设置柱状图中每个柱子的宽度，但现在g2plot似乎没有相关api。并且想在获得chart实例后如new RadialBar()后使用G2的方法，也无法成功设置宽度。
